src/collective/fingerpointing/subscribers/registry_logger.py

```diff
@@ -23,6 +23,6 @@ def registry_logger(event):
 
         if IRecordModifiedEvent.providedBy(event):
             action = 'modify'
-            extras = 'object={0} value={1}'.format(event.record, event.record.value)
+            extras = u'object={0} value={1}'.format(event.record, event.record.value)
```

Comment:
@frisi thank you for the code review.

The problem here is that you are mixing an ``unicode`` string with ``utf-8`` code.

```
In [23]: st=u'Ümläut'.encode('utf-8')
In [25]: st
Out[25]: '\xc3\x9cml\xc3\xa4ut'
In [26]: chardet.detect(st)
Out[26]: {'confidence': 0.7525, 'encoding': 'utf-8'}
In [27]: test = u'Foo {}'.format(st)
In [28]: test
Out[28]: u'Foo \xdcml\xe4ut'
In [29]: chardet.detect(test)
---------------------------------------------------------------------------
UnicodeDecodeError                        Traceback (most recent call last)
<ipython-input-29-b0a301db8b27> in <module>()
----> 1 chardet.detect(test)

/usr/lib/python2.7/dist-packages/chardet/__init__.pyc in detect(aBuf)
     22     u = universaldetector.UniversalDetector()
     23     u.reset()
---> 24     u.feed(aBuf)
     25     u.close()
     26     return u.result

/usr/lib/python2.7/dist-packages/chardet/universaldetector.pyc in feed(self, aBuf)
    113                 self._mCharSetProbers = [MBCSGroupProber(), SBCSGroupProber(), Latin1Prober()]
    114             for prober in self._mCharSetProbers:
--> 115                 if prober.feed(aBuf) == constants.eFoundIt:
    116                     self.result = {'encoding': prober.get_charset_name(),
    117                                    'confidence': prober.get_confidence()}

/usr/lib/python2.7/dist-packages/chardet/charsetgroupprober.pyc in feed(self, aBuf)
     57             if not prober: continue
     58             if not prober.active: continue
---> 59             st = prober.feed(aBuf)
     60             if not st: continue
     61             if st == constants.eFoundIt:

/usr/lib/python2.7/dist-packages/chardet/sjisprober.pyc in feed(self, aBuf)
     65                 if i == 0:
     66                     self._mLastChar[1] = aBuf[0]
---> 67                     self._mContextAnalyzer.feed(self._mLastChar[2 - charLen :], charLen)
     68                     self._mDistributionAnalyzer.feed(self._mLastChar, charLen)
     69                 else:

/usr/lib/python2.7/dist-packages/chardet/jpcntx.pyc in feed(self, aBuf, aLen)
    143         i = self._mNeedToSkipCharNum
    144         while i < aLen:
--> 145             order, charLen = self.get_order(aBuf[i:i+2])
    146             i += charLen
    147             if i > aLen:

/usr/lib/python2.7/dist-packages/chardet/jpcntx.pyc in get_order(self, aStr)
    174         if not aStr: return -1, 1
    175         # find out current char's byte length
--> 176         if ((aStr[0] >= '\x81') and (aStr[0] <= '\x9F')) or \
    177            ((aStr[0] >= '\xE0') and (aStr[0] <= '\xFC')):
    178             charLen = 2

UnicodeDecodeError: 'utf8' codec can't decode byte 0x81 in position 0: invalid start byte
```

But yes, ``safe_unicode`` utility can fix if we have this case with ``utf-8`` string in, but I'm not sure if we will eventually have this problem one day.

For me I would keep the way it is now and wait  someone show a real use case where this problem happen.